doc/release/RELEASE-NOTES.md

```diff
@@ -22,6 +22,7 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 * [OSDEV-1492](https://opensupplyhub.atlassian.net/browse/OSDEV-1492) - Fixed an issue where invalid manually entered dates were not validated on the UI, resulting in API errors with message “The request query is invalid.” on `Moderation Queue` page. Invalid dates are now trimmed and properly handled.
 
 ### What's new
+* [OSDEV-1383](https://opensupplyhub.atlassian.net/browse/OSDEV-1383) - Edited text of the automated email that notifies a contributor when one of their facilities has been claimed.
```

Comment:
Hi @FrancescaOSH,
Could you please provide a brief explanation of why you made this update? Specifically, what value do these changes bring?

Reply:
Hi @vladsha-dev I have updated the text in the release notes. Let me know if that's enough.

Reply:
Nice, thank you!